ocaml/xapi/import.ml

```diff
@@ -635,11 +635,7 @@ module VM : HandlerTools = struct
         log_reraise
           ("failed to create VM with name-label " ^ vm_record.API.vM_name_label)
           (fun value ->
-            let vm =
-              Xapi_vm_helpers
-              .create_from_record_without_checking_licence_feature_for_vendor_device
-                ~__context rpc session_id value
-            in
+            let vm = Client.VM.create_from_record ~rpc ~session_id ~value in
```

Comment:
Previously, some values were overridden:
```ocaml
~value:{r with API.vM_suspend_VDI= Ref.null; API.vM_power_state= `Halted}
```
Shouldn't this still be done? Both here and in xapi_vm_snapshot

Reply:
That override came from 07b0fd4d509825277d44258da29de37d1d758aff.

Reply:
Yes, will update